src/model/Clearing.java

```diff
@@ -28,4 +30,13 @@ public void setDwelling(String dwelling) {
         this.dwelling = dwelling;
     }
 
+    public boolean canChangeClearing(Clearing clearing) {
+        // if they're connected
+        if (Arrays.asList(connections).contains(clearing)) {
```

Comment:
not sure if we should compare names of clearings here instead of the clearing itself